config/samples-hub/hub.kmm.sigs.x-k8s.io_managedclustermodules.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: hub.kmm.sigs.x-k8s.io/v1beta1
+kind: ManagedClusterModule
+metadata:
+  name: managedclustermodule-sample
+spec:
+  jobNamespace: default
```

Comment:
Didn't we remove that field from the API?

Reply:
We did indeed. Good catch @ybettan, fixed - thanks!